eve/workers/cosbench/workload.short.xml.tmpl

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<workload name="workload1" description="Metadata (0 byte objects) sample benchmark for s3">
+    <storage type="s3" config="accesskey={{ACCESS_KEY}};secretkey={{SECRET_KEY}};endpoint={{ENDPOINT}};path_style_access=true" />
+    <workflow>
+        <!-- Initialization: Create 2 buckets -->
+        <workstage name="init">
+        <work type="init" workers="1" config="containers=r(1,1)" />
+        </workstage>
+
+        <!-- Create 0 Byte Objects -->
+        <workstage name="prepare">
+            <work type="prepare" workers="512" config="containers=r(1,1);objects=r(1,100000);sizes=c(0)MB;" />
+        </workstage>
+
+        <!-- Write 0 Byte -->
+        <workstage name="write_0B_1T">
+            <work name="write_0B_1T" workers="16" runtime="300" division="object">
+                <operation type="write" ratio="100" config="sizes=c(0)MB;containers=r(1,1);objects=r(1,100000);"/>
+            </work>
+        </workstage>
+        <workstage name="write_0B_256T">
+            <work name="write_0B_256T" workers="256" runtime="300" division="object">
+                <operation type="write" ratio="100" config="sizes=c(0)MB;containers=r(1,1);objects=r(1,100000);"/>
+            </work>
+        </workstage>
+
+        <!-- Read 0 Bytes -->
+        <workstage name="read_0B_1T">
```

Comment:
I would also suggest changing the workers to 64 and 512.
@vrancurel  What do you think?

Reply:
Can we do 1, 8, 32, 64, 128, 256, 512 ?

Reply:
@vrancurel As of right now, `Eve` can't run long enough to test all those scenarios.

Reply:
@vrancurel There's a time limit of 1 hour on the CI so with the setup, tests, and 2 stages we already get very close to this limit which is why we decided for only 2 stages to start with. It's currently a limitation of Eve's access to the GKE cluster that I believe releng is working on.

Reply:
@vrancurel The long workload I made is in `workload.xml.tmpl`. it uses 1,2,4,8,16,32,64,128,256,512,1024. When we get it setup with a proper metalk8s and Zenko deploy, we'll be able to switch.